src/data/HeaderContent.js

```diff
@@ -31,17 +31,13 @@ const Header = () => {
         Benvenuto su Jewelry Shop Olivia Rossi!
       </p>
       <Router>
-        <Route exact path='/login'>
-          <Login />
-        </Route>
+        <Routes>
+          <Route exact path='/login' element={<Login />} />
```

Comment:
The error I was seeing in the browser console:

```
Uncaught Error: [Login] is not a <Route> component. All component children of <Routes> must be a <Route> or <React.Fragment>
```

That's saying that we can never nest anything under `<Routes>` apart from a `<Route>` component.

According to the upgrade guide: [https://reactrouter.com/en/main/upgrading/v5#advantages-of-route-element](https://reactrouter.com/en/main/upgrading/v5#advantages-of-route-element)

From v5 to v6 they changed the syntax from where you nest your component under the `<Route>` to where you pass it in an `element` property instead:

```javascript
# before
<Route path='/my-page'>
	<MyPage someVariable={123} />
</Route>

# after
<Route path='/my-page' element={<MyPage someVariable={123} />} />
```

Reply:
Thank you a lot for this updated syntax, it slipped my mind 😊